src/crimson/os/seastore/transaction_manager.h

```diff
@@ -1023,7 +1024,8 @@ class TransactionManager : public ExtentCallbackInterface {
 	remap_laddr,
 	remap_length,
 	intermediate_key,
-	intermediate_base);
+	intermediate_base,
+	false);
     }
     return fut.si_then([remap_laddr, remap_length, remap_paddr](auto &&ref) {
       assert(ref->get_key() == remap_laddr);
```

Comment:
For `alloc_remapped_extent()` path, can we take the chance to introduce an optional `clone_info_t` with `intermediate_key` and `intermediate_base` to make it explict about the logic with/without clone ?